dyo/dyo.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
```

Comment:
Any reason why are you picked urfave instead of cobra?

Reply:
Seemed much more straightforward, and simple compared to cobra. Cobra is a good candidate if you want to build a spaceship, I'd like to take things more lightly if it's not necessary.